test/CMakeLists.txt

```diff
@@ -181,10 +181,7 @@ if(LINUX) # OS-specific functions are implemented only for Linux now
         LIBS ${UMF_UTILS_FOR_TEST} ${LIBNUMA_LIBRARIES})
 endif()
 
-# TODO add support for Windows
-if(UMF_BUILD_GPU_TESTS
-   AND UMF_BUILD_LEVEL_ZERO_PROVIDER
-   AND LINUX)
+if(UMF_BUILD_GPU_TESTS AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
```

Comment:
pls update Readme file (with info it works on both OSes, not only on Linux)

Reply:
Okey